pkg/util/rowcodec/encoder.go

```diff
@@ -239,7 +239,10 @@ func (NoChecksum) encode(encoder *Encoder, buf []byte) ([]byte, error) {
 	return encoder.toBytes(buf), nil
 }
 
-const checksumVersionRaw byte = 1
+// introduced since v7.1.0
+// bumped to 1 since v8.3.0
+// bumped to 2 since v8.4.0
+const checksumVersionRaw byte = 2
```

Comment:
Is it better to use three constants instead and describe what's changed in the comment for each of them?